{
  ersGeneral=new ExceptionRecognizerGeneral(Predicates.<Throwable>alwaysTrue());
  assertThat(ersGeneral.recognize(new FooException()),is("foo"));
}
